invenio_app_ils/ui/backoffice/src/pages/ItemList/ItemTable/ItemTable.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
```

Comment:
Just remember that all this will probably disappear replaced by the `invenio-search` components that we will start to develop today.

Reply:
No problem instead of manually rendering a table we will just swap components and feed invenio-search with the data.